README.md

```diff
@@ -131,10 +131,10 @@ The RAPIDS versions for things like container images and install instructions ar
 ```python
 versions = {
     "stable": {
-        "rapids_container": "nvcr.io/nvidia/rapidsai/base:25.06-cuda12.8-py3.12",
+        "rapids_container": "nvcr.io/nvidia/rapidsai/base:25.10-cuda12.8-py3.12",
```

Comment:
Do we want to update this automation to use the new container image name?

```suggestion
        "rapids_container": "nvcr.io/nvidia/rapidsai/base:25.10-cuda12-py3.12",
```

cc @bdice

Reply:
Yes. We want to advertise the new major version scheme because it is all we will ship for 25.12+.

I just merged the RSN and it will be live shortly. https://github.com/rapidsai/docs/pull/687